open-metadata-implementation/access-services/data-engine/data-engine-api/src/main/java/org/odpi/openmetadata/accessservices/dataengine/model/Attribute.java

```diff
@@ -27,6 +27,7 @@
     private String defaultValueOverride;
     private String dataType;
     private String defaultValue;
+    private String anchorGUID;
```

Comment:
Why is anchorGUID on the API?

Reply:
Thank you for spotting the issue. Now I'm using the value.